src/libs/filters/aperture.c

```diff
@@ -67,7 +67,19 @@ static gboolean _aperture_update(dt_lib_filtering_rule_t *rule)
 
 static gchar *_aperture_print_func(const double value, const gboolean detailled)
 {
-  return g_strdup_printf("%s%.1lf", detailled ? "f/" : "", value);
+  if (detailled)
+  {
+    return g_strdup_printf("%s%.1lf", detailled ? "f/" : "", value);
```

Comment:
You can then probably remove `detailled` check here. But I'm wondering why we don't need a local switch in detailled mode?

Reply:
The `detailled` is used when the list is filled with all available aperture values (right click in the text field).

<img width="199" alt="Bildschirmfoto 2024-01-06 um 12 22 06" src="https://github.com/darktable-org/darktable/assets/4083281/98c5576d-b402-4bc5-b472-bb7cd574a4ca">

Reply:
In this case it is only the textural representation, no values for computing.